pkg/deploy/controller/deployment/controller.go

```diff
@@ -107,6 +107,11 @@ func (c *DeploymentController) Handle(deployment *kapi.ReplicationController) er
 			// Don't try and re-create the deployer pod.
 			break
 		}
+
+		if _, ok := deployment.Annotations[deployapi.DeploymentIgnorePodAnnotation]; ok {
```

Comment:
Can you talk more about this? Callers are expected to manually run deployers? Can you add an example using it? Will we automate setting the annotation for users (maybe a `oc deploy` subcommand) ?